### DISCUSSION TITLE
How can I configure maxBuffer size?

### DISCUSSION THREAD

**Discussion by arahatashun** [renovatebot/renovate#27287] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest (37.187.2)

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

How can I configure maxBuffer size? I encountered the following artifact update problem:

```
stdout maxBuffer exceeded
```

and want to fix by making maxBuffer size larger.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by viceice** [comment]
not possible yet

**Reply by viceice** [reply]
https://github.com/renovatebot/renovate/blob/0867de846a371dc98e72be2245d66e4b8ef706c0/lib/util/exec/common.ts#L61

it's 10MB by default. which manager is causing the issue?

we can probably add an experimental env to allow overriding the default buffer size

**Reply by arahatashun** [reply]
@viceice 
I am managing maven dependencies using bazel.
